docs/src/main/asciidoc/kafka/kafka_overview.adoc

```diff
@@ -1016,3 +1016,56 @@ public KafkaBinderHealth kafkaBinderHealthIndicator() {
     };
 }
 ```
+
+[[check-if-the-connection-to-the-brokers-is-in-error-or-in-topic]]
+=== Check if the connection to the brokers is in error or in topic
+
+To help identify if the cluster brokers are down or if it is just a specific topic that is having a connection problem,
+in complex scenarios where the service connects with several topics this verification helps to identify the severity of the problem.
```

Comment:
Simply state that `Here is the pseudo-code for writing a custom Kafka binder HealthIndicator. In this example, we try to override the binder provided Kafka HealthIndicator by specifically checking first for cluster connectivity and then followed by topic-related issues.`